src/services/gta_data/yim_fipackfile.cpp

```diff
@@ -42,6 +63,13 @@ namespace big
 					if (encryption_type == 0xFFFFFF9)
 						continue; // skip AES encrypted RPFs
 
+					if (encryption_type == 0x4E45504F || encryption_type == 0x50584643)
+					{
+						LOG(INFO) << "OpenIV RPF, skipping " << reinterpret_cast<const char*>(file.u8string().c_str());
```

Comment:
Weird space in the end, why say OpenIV when can probably use CodeWalker to get the same result, i would name this: "Open encryption found, skipping" (Not sure what the second hash is) or: "Modified RPF found, skipping".

Reply:
Actualy I used CodeWalker to look at certain world models. But why CodeWalker "modified" files - this what should keep in mind.
[07-09-2023-10-54-19_cout.log](https://github.com/YimMenu/YimMenu/files/11994660/07-09-2023-10-54-19_cout.log) from dev build above that fixes the issue

Reply:
> Actualy I used CodeWalker to look at certain world models. But why CodeWalker "modified" files - this what should keep in mind. [cout.log](https://github.com/YimMenu/YimMenu/files/11994660/07-09-2023-10-54-19_cout.log) from dev build above that fixes the issue

What is `Launcher\Launcher.rpf`?